launch/tier4_vehicle_launch/launch/vehicle.launch.xml

```diff
@@ -5,14 +5,15 @@
   <arg name="sensor_model" description="sensor model name"/>
   <arg name="vehicle_id" default="$(env VEHICLE_ID default)" description="vehicle specific ID"/>
   <arg name="initial_engage_state" default="false" description="/vehicle/engage state after starting Autoware"/>
+  <arg name="config_dir" default="$(find-pkg-share $(var sensor_model)_description)/config" description="path to dir where sensors_calibration.yaml, etc. are located"/>
```

Comment:
Do we have to add this line as not a temporary variable but an argument?

Reply:
I think we sometimes need to change this path to an individual product's path like this https://github.com/tier4/autoware_individual_params.jpntaxi/tree/main/individual_params/config/1/aip_xx1.

Reply:
Could you give me more details about "like this". Do you mean there is a vehicle_id in the config path?

Also, in vehicle.xacro, there is not config_dir variable. Are you going to add conig_dir in vehicle.xacro?

Reply:
Sorry I'll explain it here. I'd like to take [sample_vehicle](https://github.com/autowarefoundation/sample_vehicle_launch/tree/main/sample_vehicle_description) and [sample_sensor_kit](https://github.com/autowarefoundation/sample_sensor_kit_launch) as the example. [tier4_vehicle_launch/vehicle.xacro(https://github.com/autowarefoundation/autoware.universe/blob/main/launch/tier4_vehicle_launch/urdf/vehicle.xacro) is the entrypoint for robot_description, and this file includes the urdf of the vehicle and the sensor, namely [sample_vehicle_description/vehicle.xacro](https://github.com/autowarefoundation/sample_vehicle_launch/blob/main/sample_vehicle_description/urdf/vehicle.xacro) and [sample_sensor_kit_description/sensors.xacro](https://github.com/autowarefoundation/sample_sensor_kit_launch/blob/main/sample_sensor_kit_description/urdf/sensors.xacro).
The `config_dir` variable is defined in sensors.xacro which is `sample_sensor_kit_description/config/sensors_calibration.yaml` by default ([this file](https://github.com/autowarefoundation/sample_sensor_kit_launch/blob/main/sample_sensor_kit_description/config/sensors_calibration.yaml)). So in normal case we do not need to pay attention to this.

But in our internal product this config_dir points to <vehicle_id>'s one as written in [autoware_launch launch file](https://github.com/tier4/autoware_launch.xx1/blob/a58e9ee52c233b62912a2eed1f0f4034baa6211b/vehicle_launch/launch/vehicle_description.launch.xml#L9-L12).

So if we use tier4_vehicle_launch for both normal and product cases we need an option to overwrite this `config_dir` variable depending on the type of `autoware_launch`, so we'll need to pass its value externally. I'm afraid otherwise `tier4_vehilce_launch/vehicle.launch.xml will need to resolve the path to *vehicle_id*'s individual_params.